packages/descendants/src/Descendants.types.ts

```diff
@@ -0,0 +1,12 @@
+import { ComponentProps } from 'react';
+
+/** */
+export interface Descendant<E extends HTMLElement = HTMLElement> {
+  element: E; // Could also be a RefObject
```

Comment:
i don't have a preference on element or element.current, although element.current makes it feel more like an interactive element? but that might be my own bias

Reply:
IMO there's not really a benefit to storing the `ref` here vs the element, and it just makes accessing the element a bit more verbose:

```
const firstEl  = descendants[0].element
// vs
const firstEl  = descendants[0].elementRef.current
```

Reply:
happy to leave as-is